packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md

```diff
@@ -40,9 +38,9 @@ tenant has its own security policies, user registration settings, user groups,
 roles, and rules that determine application access. An illustration of these
 different types of data is shown below:
 
-<div class="image-at-70">
+<div class="full">
```

Comment:
I think this one could be 3/4

Reply:
Agreed, and changed in the next commit. Thanks.